File: Archive/mad/prg/Exercises/Rt_SpaceInvader/SpaceInvaders.java
import java.util.Scanner;

public class SpaceInvaders{
    public static void main(String args[]){
        Scanner in = new Scanner(System.in);
        int cases = in.nextInt();
        while (cases-- > 0){
            int records = in.nextInt();
            int doubleRecords = records * 2;
            StringBuilder output = new StringBuilder();
            output.setLength(records * 3);
            for (int i = 0; i < records; i++){
                String name = in.next();
                output.setCharAt(i, name.charAt(0));
                output.setCharAt(i + records, name.charAt(1));
                output.setCharAt(i + doubleRecords, name.charAt(2));
            }
            System.out.println(output);
        }
        in.close();
    }
}

File: Archive/mad/prg/Exercises/Rt_SpaceInvader/SpaceInvaders.c
#include <stdio.h>
#include <stdlib.h>

// Esta weá funciona, pero por algún motivo no lo coge Acepta el Reto

int main(){
    int cases;
    scanf("%d", &cases);
    while (cases-- > 0){
        int records;
        scanf("%d", &records);
        int doubleRecords = records * 2;
        char* output = (char*) malloc(records * 3);
        int i = 0;
        while (i < records){
            char name[] = "___";
            scanf("%s", &name);
            output[i] = name[0];
            output[i + records] = name[1];
            output[i + doubleRecords] = name[2];
            i++;
        }
        printf("%s", output);
        free(output);
    }
    return 0;
}